#!/usr/bin/env python
PACKAGE = "accelerometer_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("window_size", int_t, 0, "window_size", 20, 2,100)

exit(gen.generate(PACKAGE, "accelerometer_ros", "accelerometerGauss"))
